src/components/Form/EyeColor/EyeColor.jsx

```diff
@@ -139,7 +139,7 @@ export default class EyeColor extends ValidationElement {
                  onBlur={this.props.onBlur}
                  onFocus={this.props.onFocus}
                  >
-            <div className="eye-icon">
+            <div className="eye-icon hazel">
```

Comment:
From the [comment](https://github.com/18F/e-QIP-prototype/pull/16#issuecomment-278813101) below:

> The SVGs, from what I was told, needed to be inline in order to manipulate them with CSS 3 transitions, adjusting colors, stroke widths, etc.

When we do a bit of refactoring to hopefully centralize these use cases I am optimistic some will be found to not need to be referenced in this manner.